apps/src/lib/util/firehose.js

```diff
@@ -281,6 +292,20 @@ class FirehoseClient {
   }
 }
 
+// Verifies that given data will not fail firehose batch
+function validateFirehoseDataSize(data) {
+  const json_size = new Blob([data?.data_json]).size;
```

Comment:
The '?' on this line and the one below are additions over the prior PR

Reply:
Not necessary- the size returns '9' which I suppose is for the array storage of a null value.